SparkPost/2web2ui
# TR-1286: Basic editing for Prototype fullscreen code editor with preview

This is a subtask [TR-1286](https://jira.int.messagesystems.com/browse/TR-1286) of [TR-1204](https://jira.int.messagesystems.com/browse/TR-1204).

### What Changed
- Set title to template name
- Added tabs (can deep link to a tab)
- Dark mode editor with save as draft button

<img width="1389" alt="Screen Shot 2019-05-17 at 12 24 08 PM" src="https://user-images.githubusercontent.com/1335605/57942130-b0ed0500-789e-11e9-8a13-d816f4befbf5.png">


### How To Test
- Visit http://app.sparkpost.test/templates/edit/amp/next
- Click a tab (should update url)
- Modify content
- Click "Save As Draft" (should disable button while processing)
- Refresh page (should have saved most recently saved changes)
